src/wordlift/mappings/pages/class-edit-mappings-page.php

```diff
@@ -242,12 +250,78 @@ private static function get_post_type_key_and_value() {
 				)
 			);
 		}
+
 		return array(
 			'post_type_option_name' =>  $post_type_option_name,
 			'post_type_option_values' => $post_type_option_values
 		);
 	}
 
+    /**
+     * Return post type option and post type option values.
+     *
+     * @return array Array of post_type_option and post_type_option_values.
+     */
+    private static function get_post_taxonomy_key_and_value() {
```

Comment:
We need to make taxonomy and terms load asynchronously, this is necessary because there can be a lot of taxonomy terms which will slow down the ui.